components/browser/domains/README.md

```diff
@@ -7,9 +7,15 @@ Localized and customizable domain lists for auto-completion in browsers.
 ### Setting up the dependency
 
 Use gradle to download the library from JCenter:
-
+In root build.gradle file:
+```groovy
+maven {
+   url "maven.mozilla.org"
+}
```

Comment:
Lets only add this to the root readme file, adding a section above the [Components](https://github.com/mozilla-mobile/android-components#components) section, can be a good option :slightly_smiling_face:

The url should be  "https://maven.mozilla.org/maven2" instead of "maven.mozilla.org"

Reply:
I have added this in Root and also updated the URL in all the components Read Me files.

Reply:
Great!
But just let keep it in the main README file, it's easier to maintain afterwards!

Reply:
So, should I remove this from components' read me files? Or let them be there?

Reply:
Yes, please.
If it's a lot of work I can help you with that.
Just let me know.

Reply:
No problem. Give me few minutes, I am on it.

Reply:
Done